tmp/axum-wasm-expanded/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "shuttle-axum-expanded"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = [ "cdylib" ]
+
+[dependencies]
+# most axum features can be enabled, but "tokio" and "ws" depend on socket2
+# via "hyper/tcp" which is not compatible with wasi
+axum = { version = "0.6.0", default-features = false }
+futures-executor = "0.3.21"
+http = "0.2.7"
+tower-service = "0.3.1"
+rmp-serde = { version = "1.1.1" }
+
+[dependencies.shuttle-common]
+path = "../../common"
+default-features = false
```

Comment:
I removed the default feature from common btw, since we can't set `default-features = false` in inherited deps: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace